x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/rest/RestDeleteRollupJobAction.java

```diff
@@ -19,12 +21,20 @@
 
 import java.io.IOException;
 
+import static org.elasticsearch.rest.RestRequest.Method.DELETE;
+
 public class RestDeleteRollupJobAction extends BaseRestHandler {
+
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(LogManager.getLogger(RestDeleteRollupJobAction.class));
+
     public static final ParseField ID = new ParseField("id");
 
     public RestDeleteRollupJobAction(Settings settings, RestController controller) {
         super(settings);
-        controller.registerHandler(RestRequest.Method.DELETE, Rollup.BASE_PATH +  "job/{id}/", this);
+        // TODO: remove deprecated endpoint in 8.0.0
+        controller.registerWithDeprecatedHandler(
+                DELETE, "/_rollup/job/{id}", this,
+                DELETE, Rollup.BASE_PATH +  "job/{id}/", deprecationLogger);
```

Comment:
I may be reading this wrong, but I think the deprecated path (`Rollup.BASE_PATH +  "job/{id}/"`) will resolve to `/_rollup/job/{id}` because `BASE_PATH` itself was changed above to no longer include `_xpack`?

Reply:
Oops, that was an overzealous find/replace. That one should not have been changed.

Reply:
I pushed 146acddac12249ac67d0fb1e97c318088a50b181.

Reply:
I pushed 146acddac12249ac67d0fb1e97c318088a50b181.

Reply:
:+1: